lib/flux/app-state.ts

```diff
@@ -161,9 +139,7 @@ export const actionMap = new ActionMap({
               modificationDate: timestamp,
               shareURL: '',
               publishURL: '',
-              tags: ([] as T.TagName[]).concat(
```

Comment:
I had no idea that if you created a new note while on a tag that it would add that tag. I think we should delete that and add it back as we move newNote over to Redux.

Reply:
the idea originally was that since you were "inside" the tag you would be creating a note inside of it too. indicating which tab we're inside has always been awkward. I'd be more inclined to keep the behavior until we make a decision to change it because otherwise we're going to have to come back and undo this refactor if we don't like the behavioral change.

is it difficult to maintain the existing behavior?

Reply:
Added back, done.